repositories/SectionRepository.test.js

```diff
@@ -56,17 +57,20 @@ describe("SectionRepository", () => {
       questionnaire: { id: questionnaireId }
     } = await setup();
 
-    const result = await SectionRepository.insert(
+    const section = await SectionRepository.insert(
       buildSection({ questionnaireId: questionnaireId })
     );
 
-    await SectionRepository.update({
-      id: result.id,
-      title: "updated title"
-    });
-    const updated = await SectionRepository.getById(result.id);
+    const update = {
+      id: section.id,
+      title: "updated title",
+      alias: "updated alias"
+    };
+
+    await SectionRepository.update(update);
+    const result = await SectionRepository.getById(section.id);
 
-    expect(updated.title).not.toEqual(result.title);
+    expect(result).toEqual(expect.objectContaining(update));
```

Comment:
Can we use `toMatchObject`?

Reply:
Yes. This works despite the misleading name.